airflow/providers/amazon/aws/example_dags/example_eks_with_nodegroup_in_one_step.py

```diff
@@ -36,11 +36,15 @@
     'endpointPrivateAccess': False,
 }
 
+default_args = {
+    'cluster_name': CLUSTER_NAME,
+}
 
 with DAG(
     dag_id='example_eks_using_defaults_dag',
+    default_args=default_args,
```

Comment:
Can you add `default_args={'cluster_name': CLUSTER_NAME},` directly here as well?

Reply:
Should be addressed in https://github.com/apache/airflow/pull/18645/commits/b577f159075d1c6b75d90c54303041b78025f122